library/alloc/src/boxed.rs

```diff
@@ -1030,11 +1030,11 @@ where
 }
 
 #[stable(feature = "box_from_slice", since = "1.17.0")]
-impl<T: Copy> From<&[T]> for Box<[T]> {
+impl<T: Clone> From<&[T]> for Box<[T]> {
     /// Converts a `&[T]` into a `Box<[T]>`
     ///
     /// This conversion allocates on the heap
-    /// and performs a copy of `slice`.
+    /// and performs a clone of `slice`.
```

Comment:
Changing the bounds here is insta-stable. Let's not do this for now, and remember to change this as part of the stabilization later.